packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { StoryObj, Meta } from "@storybook/react";
+import Tooltip from "./Tooltip";
+import { useRef, useState } from "react";
+
+const TooltipInteractive = () => {
+  const inputElement = useRef<HTMLButtonElement>(null);
+  const [isHoverOpened, setIsHoverOpened] = useState(false);
+
+  return (
+    <>
+      <button
+        ref={inputElement}
+        onMouseEnter={() => setIsHoverOpened(true)}
+        onMouseLeave={() => setIsHoverOpened(false)}
+      >
+        the button
+      </button>
+      <Tooltip
+        placement="bottom"
+        referenceEl={inputElement}
+        open={isHoverOpened}
+      >
+        {"Some long long long long long long text in tooltip"}
+      </Tooltip>
+    </>
+  );
+};
+
+export default {
+  title: "Floating-Components/Tooltip",
+  component: Tooltip,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1007-4109&t=1jAyqXTkYBP57oZL-0",
+    },
+  },
+  argTypes: {
+    string: {
+      control: "text",
```

Comment:
@JFChoe could we also add args for `open`, `placement` and make sure that updating those work in Storybook?

Reply:
@JFChoe for instance, updating `string` doesn't update the tooltip
![image](https://user-images.githubusercontent.com/127199/234412866-b9fb298e-983d-4a7b-9ec8-fee308ee5748.png)

Reply:
I added in a select list for the `Placement` prop